workbench-core/example/infrastructure/integration-tests/support/setup.ts

```diff
@@ -29,9 +29,7 @@ export default class Setup {
     return session;
   }
 
-  public async createAdminSession(): Promise<ClientSession> {
-    const userPoolId = this._settings.get('ExampleCognitoUserPoolId');
-    const clientId = this._settings.get('ExampleCognitoUserPoolClientId');
+  public async createClientSession(userPoolId: string, clientId: string): Promise<ClientSession> {
     const rootUserNameParamStorePath = this._settings.get('rootUserNameParamStorePath');
     const rootPasswordParamStorePath = this._settings.get('rootPasswordParamStorePath');
```

Comment:
This info is from an admin user, so renaming it from `createAdminSession` to a generic `createClientSession` seems misleading to me.

Reply:
Renamed to `createRootUserSession` to help clarify the intention